src/components/signin/Mnemonics.js

```diff
@@ -26,6 +26,8 @@ const Mnemonics = ({ screenProps, navigation, styles }) => {
   const [showDialog] = useDialog()
   const [showErrorDialog, hideDialog] = useErrorDialog()
 
+  AsyncStorage.removeItem('web3Token')
```

Comment:
Why do we need it here?
You remove web3Token in many places. why?

Reply:
If user will try to go back to home page he will be redirected to recover again as there is functionality which checks if there is we3Token in AsyncStorage